.eslintignore

```diff
@@ -1,2 +1,3 @@
 dist
 node_modules
+legacy
```

Comment:
this is just eslint ignores. Because the old code used a different rule set (that was far more complicated, btw. I switched to airbnb with just a few tweaks instead), and I didn't want to introduce a bunch of changes to that code in the git history.